packages/grafana-data/src/types/datasource.ts

```diff
@@ -558,6 +558,10 @@ export interface DataSourceSettings<T extends DataSourceJsonData = DataSourceJso
   access: string;
   url: string;
   user: string;
+  /**
+   *  @deprecated -- use jsonData to store information related to database.
+   *  This field should only be used by Elasticsearch and Influxdb.
```

Comment:
Sorry if it was asked somewhere else already, but are we planning to remove the use of this from Elasticsearch and Influxdb as well?

Reply:
Sorry but I don't know anything about them. Maybe @grafana/observability-logs can answer?

Reply:
we'd like to remove the use of this from Elasticsearch, but there's nothing planned yet (i created a github-issue to track this at https://github.com/grafana/grafana/issues/59098)

Reply:
regarding Influxdb, pinging @grafana/observability-metrics 😄

Reply:
I don't know of any plan to deprecate this, but it doesn't look like a huge lift to follow the plan laid out in the issue Gabor created above. I created an issue here https://github.com/grafana/grafana/issues/59115, so we don't loose track of this request.